### ISSUE TITLE
Not ignoring pluralisation

### ISSUE THREAD

**Issue by i2chris** [glebm/i18n-tasks#331]
It ignores datetime keys but the missing pluralisation keys still appear in the missing translations section.  Is there a specific rule needed to ignore pluralisations?

```
ignore_missing:
  - '{datetime, activerecord, date, datetime,i18n, number, devise, spree, support, time, helpers}.*'
```

```
Missing translations (13) | i18n-tasks v0.9.29
+--------+------------------------------------------------+----------------------------------+
| Locale | Key                                            | Value in other locales or source |
+--------+------------------------------------------------+----------------------------------+
|   pl   | cannot_register_samples.counting               | few, many                        |
|   pl   | datetime.distance_in_words.about_x_hours       | few, many                        |
|   pl   | datetime.distance_in_words.about_x_months      | few, many                        |
|   pl   | datetime.distance_in_words.about_x_years       | few, many                        |
|   pl   | datetime.distance_in_words.almost_x_years      | few, many                        |
|   pl   | datetime.distance_in_words.less_than_x_minutes | few, many                        |
|   pl   | datetime.distance_in_words.less_than_x_seconds | few, many                        |
|   pl   | datetime.distance_in_words.over_x_years        | few, many                        |
|   pl   | datetime.distance_in_words.x_days              | few, many                        |
|   pl   | datetime.distance_in_words.x_minutes           | few, many                        |
|   pl   | datetime.distance_in_words.x_months            | few, many                        |
|   pl   | datetime.distance_in_words.x_seconds           | few, many                        |
|   pl   | you_are_adding_items_from_the_trolley.counting | few, many                        |
+--------+------------------------------------------------+----------------------------------+
```

**Comment by nitsujri** [comment]
Language specific `ignore_missing` doesn't work yet: https://github.com/glebm/i18n-tasks/issues/339

**Comment by davidwessman** [comment]
@glebm Closed by #389 ?
